lib/libs/email/content/email-components.tsx

```diff
@@ -92,6 +92,38 @@ const LoginInstructions = ({
   </ul>
 );
 
+const SubDocHowToAccess = ({
+  appEndpointURL,
+}: {
+  appEndpointURL: string;
+  useThisLink?: boolean;
+}) => (
+  <>
+    <Text style={{ ...styles.text.base, fontWeight: "bold" }}>How to Access:</Text>
+    <ul>
+      <li>
+        <Text style={styles.text.description}>
+          These documents can be found in OneMAC through this link{" "}
+          <Link href={appEndpointURL}>{appEndpointURL}</Link>.
+        </Text>
+      </li>
+      <li>
+        <Text style={styles.text.description}>
+          If you are not already logged in, click “Login” at the top of the page and log in using
+          your Enterprise User Administration (EUA) credentials.
+        </Text>
+      </li>
+
+      <li>
+        <Text style={styles.text.description}>
+          After you logged in, click the submission ID number on the dashboard page to view details.
+        </Text>
+      </li>
+    </ul>
+    <Hr style={styles.divider} />
```

Comment:
Should we just use the `Divider` component here?

Reply:
On second thought, wouldn't it better to move this before everything else, and then use `Divider` in the actual email template itself (before the `<Text>Thank you.</Text>`)? I think most of our email components have the `Divider` as the first element

Reply:
Oh thank you for catching this, I moved `<Divider />` up in the components and replaced it in all places where `<Hr>` was being used. Then I also updated this component to have the `divider` at the top of the component instead of the bottom.